hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/job/TensorFlowJob.groovy

```diff
@@ -190,10 +100,7 @@ class TensorFlowJob extends HadoopJavaProcessJob {
    * @param workerGpus Number of GPUs for each TensorFlow worker
    */
   @HadoopDslMethod
-  void workerGpus(int workerGpus) {
-    this.workerGpus = workerGpus;
-    setJobProperty("worker_gpus", this.workerGpus);
-  }
+  void workerGpus(int workerGpus);
```

Comment:
We might also need amGpus? It might be easier for single node training to happen just in am as the preprocessing job?